src/pages/map/map.ts

```diff
@@ -78,4 +114,57 @@ export class MapPage {
     this.layers.push(new Marker(location.id, location.geometry.coordinates, { icon: defIcon }));
   }
 
-}
\ No newline at end of file
+  /**
+   * Center the map on the user's current position if in Onex
+   * (otherwise remain at the default position).
+   */
+  private autoCenterMap() {
+    this.getCurrentPosition().then(currentPosition => {
+      if (turf.inside(currentPosition, ONEX_BBOX)) {
+        this.centerMap(currentPosition);
+      }
+    }).catch(() => { /* do nothing */ });
```

Comment:
Why the `catch` if nothing is done there ?

Reply:
Otherwise an "unhandled Promise rejection" error is logged. Will improve the comment.